src/ol_openedx_chat/block.py

```diff
@@ -141,7 +141,8 @@ def student_view_aside(self, block, context=None):
             "learning_mfe_base_url": settings.LEARNING_MICROFRONTEND_URL,
             "drawer_payload": {
                 "blockType": block_type,
-                "title": f"about {block.display_name}",
+                # Frontend will style AskTIM slightly
+                "title": f"AskTIM about {block.display_name}",
```

Comment:
@asadali145 I didn't realize the new version of smoot wanted us to add `AskTIM` here, otherwise we could have included it earlier. Oops!